template/audit-lister.pug

```diff
@@ -0,0 +1,59 @@
+.(id='{$_id}', class='{_ui}ui{/} {$class} {_class}feed{/}', style='{$style}', {$attributes}='')
+  | {rows}
+  | {row}
+  .event
+    .label
+      i.icon(class='{icon}comment{/}')
+    .content
+      .summary
+        .user {$user}
+        |                 {$action}
+        .date {$ts}
+      .meta
+        | {$descr}
+      .extra.text
+        | {changes}
+        | {change}
+        // option 1 - show as segment with field name in header and labels inside
```

Comment:
remove comments from {change}. Template is dumb and will add LOTS of those comments into output.

Placing them between {changes} and {change} is best.

Reply:
fixed